src/server/scripts/Outland/TempestKeep/Eye/boss_kaelthas.cpp

```diff
@@ -1163,9 +1163,21 @@ class spell_kaelthas_mind_control : public SpellScript
         targets.remove_if(Acore::ObjectTypeIdCheck(TYPEID_PLAYER, false));
     }
 
+    void HandleEffect(SpellEffIndex /*effIndex*/)
+    {
+        if (!GetCaster() || !GetHitPlayer())
+            return;
+
+        if (Player* player = GetHitPlayer())
+        {
+            GetCaster()->GetThreatMgr().ResetThreat(player);
+        }
```

Comment:
I dont like the way you are mixing brackets. Don't use them for oneliners
```suggestion
        if (!GetCaster() || !GetHitPlayer())
            return;

        if (Player* player = GetHitPlayer())
            GetCaster()->GetThreatMgr().ResetThreat(player);
```

Reply:
habbit

Reply:
habit